src/main/java/com/epam/novostroinyi/api/step/LaunchesUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.novostroinyi.api.step;
+
+import static com.epam.novostroinyi.core.util.JsonUtils.readJsonSingleValue;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
```

Comment:
there is @UtilityClass in Lombok which creates private constructor and  makes the class final  
https://projectlombok.org/features/experimental/UtilityClass